js/app.js

```diff
@@ -135,6 +136,12 @@ var TacoMap = function(mapEl, database, initialPosition, initialZoom) {
     zIndex: 10
   });
 
+  if (initialPosition) {
+      this._iw.setContent($(CONFIG.TACO_MAP.initialMessageWithPos).html());
+  } else {
+      this._iw.setContent($(CONFIG.TACO_MAP.initialMessage).html());
+  }
+
```

Comment:
Minor nit, but lines 140 and 142 should be indented two spaces to match current style.